docs/examples/make_pymvpa_collection.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# A little script to prepare PyMVPA demo collection based on data used in
+# tutorials etc
+
+set -eu
+
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+
+# getting its content, since publishing can't clone the repo, we need to push
+# content to the published handle and therefore we would loose the connection
+# to content, that is not addurl'ed but just available via the original annex.
+# Note: May be therefore use "git annex get . --from=origin" to not unnecessarily
+# get things, that are reachable from the published handle (and any clone of
+# it)? Need to add that option to datalad get.
```

Comment:
@bpoldrack  just now after being merged spotted it. The trick is to add git special remote pointing to original git repo, and then autoenable it (recent annex should be able to do that but I haven't tried yet). Fetching all data to publish collection pointing to an existing repo would not scale :-)